vyper/__init__.py

```diff
@@ -10,4 +10,4 @@
 try:
     __version__ = pkg_resources.get_distribution('vyper').version
 except pkg_resources.DistributionNotFound:
-    __version__ = 'development'
+    __version__ = '0.0.0development'
```

Comment:
Can't we get this from bumpversion?

Reply:
Yes bumpversion sets it in setup.py, and setup.py sets it in pkg_resources module ;)
So if not in development (locally in your own repo), but installed it pulls in the bumpversion/setup.py.